cloudtests.py

```diff
@@ -64,10 +64,11 @@ class TestJournalWritten(unittest.TestCase):
     def test_journal_written(self):
         """Test to check if journal gets written to disk"""
 
-        out, err, eid = system("sudo ls -l /proc/$(pgrep journal)/fd/ | grep journal")
+        out, err, eid = system("sudo ls -l /proc/$(systemctl show systemd-journald.service -p MainPID | sed 's/MainPID.*=//')/fd/ | grep journal")
```

Comment:
Why don't we break it in two steps? First to get the PID, and then just use it in the ls -l command?